src/specs/models/api-body-schemas/PlotTableConfig.v1.yaml

```diff
@@ -8,24 +8,32 @@ properties:
   plotUuid:
     type: string
     description: The UUID of the plot or table.
-  type:
+  plotType:
```

Comment:
Changed to `plotType` because `type` is an AWS reserved value, throwing an error when used in `DynamoDB.updateItem`